corehq/apps/reports/views.py

```diff
@@ -162,6 +167,39 @@
 require_can_view_all_reports = require_permission(HqPermissions.view_reports)
 
 
+def _can_view_form_attachment():
+    def decorator(view_func):
```

Comment:
Since this doesn't take args anymore, you can drop an indent level and the `can_view_form_attachment = _can_view_form_attachment()` line below and just use this decorator function directly:
```suggestion
def _can_view_form_attachment(view_func):
```

Reply:
Ah, true.